### ISSUE TITLE
[feature/react-intl] Internationalized page titles

### ISSUE THREAD

**Issue by filipstefansson** [kriasoft/react-starter-kit#1007]
Before #849 we could use `injectIntl` and then `context.setTitle()` to have page titles in different languages like this:

```javascript
const messages = defineMessages({
  title: {
    id: 'title',
    defaultMessage: 'My title',
    description: 'A page title',
  },
})
...
const { formatMessage } = props.intl;
const title = formatMessage(messages.title)
context.setTitle(title);
...
export default injectIntl(Page);
```

Is there a new way to do this since we can't use `formatMessage` in for example `routes/home/index.js`, where the title is now set? 

(`formatMessage` only works inside a React component)

Something like this is what I'm looking for:
```javascript
const messages = defineMessages({
  title: {
    id: 'title',
    defaultMessage: 'My title',
    description: 'A page title',
  },
})
...
async action(context) {
  return {
    component: <Home />,
    title: formatMessage(messages.title) // <- does not work
  };
}
```

**Comment by lookis** [comment]
Same problem, any one?

**Comment by langpavel** [comment]
Oh.. ping me tomorrow.. but @filipstefansson is right, this is possible through context.intl

**Comment by langpavel** [comment]
Hi @lookis, sorry for delay.

Look at it now, it works for `/about` page for example.

**Comment by langpavel** [comment]
Hmm, this is probably not what you are looking for..

**Comment by zbrukas** [comment]
👍

**Comment by manhhailua** [comment]
The page title is currently out of a component scope, so, React-Intl does not make sense. I've been looking into this in my project.

**Comment by langpavel** [comment]
Resolved by #1346
